test/azure/src/main/java/fixtures/azureparametergrouping/ParameterGroupings.java

```diff
@@ -197,6 +242,14 @@
     /**
      * Post parameters with a shared parameter group object.
      *
+     * @throws IllegalArgumentException thrown if parameters fail the validation
+     * @return the {@link RestResponse<Void, Void>} object if successful.
+     */
+    RestResponse<Void, Void> postSharedParameterGroupObjectWithRestResponseAsync();
```

Comment:
Are you sure this is correct? A method that has no parameters and doesn't return anything?